frontend/src/javascript/utils/googleLoginUtils.js

```diff
@@ -0,0 +1,34 @@
+import { sendRequest } from "@/javascript/utils/axiosUtils";
+
+const toggleLoginButton = (show) => {
```

Comment:
To mi wygląda na kawałek kodu pisany za czasów kiedy nikt nie wiedział o frameworkach do frontendu :D. Ogólnie to proponuję stworzyć nowy store w vuex (np. auth) i tam przechowywał informację obiekt zalogowanego usera oraz flagę isLogged (ewentualnie getter () => user != null). Dzięki temu bezpośrednio na tych przyciskach można dodać atrybut `:class="{visible: isLogged}"` (to isLogged warto zmapować korzystając z funkcji vuex, w projekcie znajdziesz dużo przykładów, albo dopytaj mnie o to). Wtedy jak zmienimy tą flagę z dowolnego miejsca w aplikacji te przycisku staną się wydoczne/niewidoczne, nie trzeba się bawić w ręczne przełączanie widoczności.

Reply:
właściwie całe ta funkcja jest tylko tymczasowa, to docelowo i tak będzie gdzie indziej. Po stronie frontendu starałem się to trzymać minimalne, a zrobić to "poprawnie" można w Autentykacja użytkowników - frontend

Reply:
No ok, w porządku.